### DISCUSSION TITLE
Video Tutorials on advance js

### DISCUSSION THREAD

**Discussion by m-Jawa-d** [javascript-tutorial/en.javascript.info#3643] [General]
Video Tuts on advance js would be really helpful

**Comment by lonchanick** [comment]
What a wonderful resource but, since programming is a task that you learn by doing, would be amazing to have a series of videos with 6 o 7 projects in parallel (as homework), evolutionary projects (that increase in complexity every time one advance in topics). finally, would be nice to have the possibility to get a certification throw exam (a paid certification: I would pay till 50$ for it)
